hydropandas/extensions/plots.py

```diff
@@ -307,6 +307,247 @@ def interactive_map(
+                           color=section_well_layout_color,
+                           )
+        ax_section.scatter(plot_x,
+                           self._obj.screen_top.values,
+                           section_markersize/2,
+                           label='screen top',
+                           marker='x',
+                           color=section_well_layout_color,
+                           )
+        ax_section.scatter(plot_x,
+                           self._obj.screen_bottom.values,
+                           section_markersize,
+                           label='screen bottom',
+                           marker='+',
+                           color=section_well_layout_color,
+                           )
+
+        # loop over all wells, plot observations and details in section plot
+        for counter, name in enumerate(self._obj.index):
+            # which column to plot?
+            cols = list(cols)
+            for i, col in enumerate(cols):
+                if col is None:
+                    cols[i] = self._obj.loc[name, "obs"]._get_first_numeric_col_name()
+            
+            # create plot dataframe
+            plot_df = self._obj.loc[name, "obs"][tmin:tmax][cols].copy()
+            if plot_df.empty or plot_df[cols].isna().all().all():
+                logger.warning(f"{name} has no data between {tmin} and {tmax}")
+                continue
+            
+            # PART 1: plot timeseries of observations
+            # one or multiple columns to plot?
+            if len(cols) == 1:
+                p = ax_obs.plot(plot_df, label=name)
+            else:
+                for col in cols:
+                    p = ax_obs.plot(plot_df[col], label=f'{name}, {col}')
+
+            if check_obs_close_to_screen_bottom:
+                # add horizonal line to plot when minimum observation in first plot column is close to bottom of screen
+                offset = 0.1
+                if (self._obj.loc[name, "screen_bottom"] > (plot_df[cols[0]].dropna().min()-offset)):
+                    ax_obs.axhline(y=self._obj.loc[name, "screen_bottom"],
+                                   ls='--',
+                                   lw=4,
+                                   alpha=0.5,
+                                   label=f'screen bottom of {name}\nis close to minimum observation',
+                                   color=p[0].get_color()
+                                   )
+
+            # PART 2: fancy section plot with lines along tube
+
+            # highlight filter on section plot
+            ax_section.plot([plot_x[counter]]*2,
+                            [self._obj.loc[name, "screen_top"], self._obj.loc[name, "screen_bottom"]],
+                            color='k',
+                            lw=3,
+                            ls=':',
+                            )
+
+            # highlight blind tube on section plot
+            ax_section.plot([plot_x[counter]]*2,
```

Comment:
use `zorder` to ensure the markers for statistics and tube characteristics appear above the lines

Reply:
Solved in update.